config/routes/api_entreprise.rb

```diff
@@ -20,23 +20,21 @@
     get '/compte/deconnexion', to: 'sessions#destroy', as: :logout
     get '/compte/apres-deconnexion', to: 'sessions#after_logout', as: :after_logout
 
-    get '/compte', to: 'users#profile', as: :user_profile
+    get '/compte', to: 'authorization_requests#index', as: :homepage
```

Comment:
Je ne comprenais pas trop ce renaming dans les vues, je pensais que c'était un changement de comportement, mais en fait non. Je ne comprends pas pourquoi `/compte` s'appelle `homepage`, la homepage c'est `/` pour moi. Pourquoi ce renaming ? 
D'ailleurs en vrai, `/compte` qui renvoi sur `authorization_requests#index` ça n'a pas trop de sens je trouve.. faudrait faire un redirect imo.

Reply:
Okay alors.
Peut être que homepage c'était pas très bien trouve comme naming.
Dans les maquettes on a défini que authorization_request#index serait la page de base quand on se log et donc sur /compte.

J'intuite fortement le fait que cette page va changer.

Du coup j'ai séparer les responsabilités :
- Dès fois on veut vraiment afficher la page de profile, genre après avoir cliquer sur les boutons de la pages de profile
- Dès fois on veut vraiment afficher la page d'index des habilitations, genre après avoir cliqué sur les boutons des la page
- Dès fois on veut rediriger vers la homepage du /compte par ex si on a pas les droits.

Du coup j'ai fais comme ca mais oui on peut : 
- 1 rediriger ce serait plus cohérent
- 2 changer le naming

Reply:
> Dans les maquettes on a défini que authorization_request#index serait la page de base quand on se log et donc sur /compte.

Mon problème c'est ça --^ ça n'a pas de sens imo.
J'ai aucun souci à ce que la page par défaut soit la liste des habilitations, et donc que ce soit `/compte/habilitations` (ou `/habilitations` btw). Pour moi `/compte` ça devrait être le profil du user (actuellement le `/compte/profile`)

Quand bien même on n'est pas d'accord sur l'url final c'est pas très grave imo (surtout que les users s'en fichent en pratique, et que c'est in-fine quasiment juste pour les personnes qui connaissent le web, donc les devs), par contre manipuler `homepage_path` dans le code et que ce ne soit pas la homepage c'est non pour moi, faut rename (en `authorization_requests_page` de ce que je comprends).

Reply:
Yes ok